mock/py/mockbuild/config.py

```diff
@@ -764,6 +759,12 @@ def list_configs(config_opts):
 @traceLog()
 def simple_load_config(name, config_path=None):
     """ wrapper around load_config() intended by use 3rd party SW """
+
+    for the_id in getresuid() + getresgid():
+        if the_id != 0:
+            continue
+        raise exception.ConfigError("Can't parse Mock configuration as root")
```

Comment:
I would argue, that this should be just a warning. If someone wants to do that. We should allow them to shoot in their leg.